sonic-chassisd/scripts/chassisd

```diff
@@ -362,6 +367,29 @@ class ModuleUpdater(logger.Logger):
         else:
             return False
 
+    def is_module_reboot_expected(self, key):
+        fvs = self.module_reboot_table.get(key)
+        if isinstance(fvs, list) and fvs[0] is True:
+            return True
```

Comment:
Since you are setting the field 'reboot' in https://github.com/sonic-net/sonic-utilities/pull/3292/files#diff-ce30e9c8fb7da57f215be6716ce0954e63528c44fc5f4ae6362adec13c48e334R210, shouldn't we be reading the value of reboot key?